src/Compilers/VisualBasic/Test/Syntax/Parser/ParseStatements.vb

```diff
@@ -8441,6 +8441,102 @@ End Module
 )
     End Sub
 
+    <Fact()>
+    <WorkItem(65, "https://github.com/dotnet/vblang/issues/65")>
+    Public Sub ParseLineContWithNoSpaceBeforeCommentV16()
+        ' PROTOTYPE LanguageVersion.Latest should be LanguageVersion.VisualBasic16 _
+        ParseAndVerify(
+        <![CDATA[
+    Module M
+           Sub Main()
+                Dim I As Integer _' Comment
+                    = 1
+            End Sub
+     End Module
+    ]]>, New VisualBasicParseOptions(LanguageVersion.Latest)
+        )
+    End Sub
+
+    <Fact()>
+    <WorkItem(65, "https://github.com/dotnet/vblang/issues/65")>
+    Public Sub ParseLineContWith1SpaceBeforeCommentV16()
+        ' PROTOTYPE LanguageVersion.Latest should be LanguageVersion.VisualBasic16 _
+        ParseAndVerify(
+        <![CDATA[
+    Module M
+           Sub Main()
+                Dim I As Integer _ ' Comment
+                    = 1
+            End Sub
+     End Module
+    ]]>, New VisualBasicParseOptions(LanguageVersion.Latest)
+        )
+    End Sub
+
+    <Fact()>
+    <WorkItem(65, "https://github.com/dotnet/vblang/issues/65")>
+    Public Sub ParseLineContWithMultipleSpacesBeforeCommentV16()
+        ' PROTOTYPE LanguageVersion.Latest should be LanguageVersion.VisualBasic16 _
+        ParseAndVerify(
+        <![CDATA[
+    Module M
+           Sub Main()
+                Dim I As Integer _   ' Comment
+                    = 1
+            End Sub
+     End Module
+    ]]>, New VisualBasicParseOptions(LanguageVersion.Latest)
+        )
+    End Sub
+
+    <Fact()>
+    <WorkItem(65, "https://github.com/dotnet/vblang/issues/65")>
+    Public Sub ParseLineContWithCommentV16()
```

Comment:
>ParseLineContWithCommentV16 [](start = 15, length = 27)

Is this a duplication of ParseLineContWith1SpaceBeforeCommentV16?

Reply:
Yes, Deleted